<!DOCTYPE HTML>
<!--
	My personal Website for Jiaming Yu, Cambridge 
	jy380@cam.ac.uk
-->
<html>
	<head>
		<title>Jiaming Yu</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" type="image/png" href="images/university-of-cambridge-logo-3.png"/>
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header" class="alt">
						<h1><a href="index.html">Jiaming Yu</a></h1>
						<nav>
							<!-- <a href="index_ch.html">中文(制作中)</a> -->
							<a href="#menu">Menu </a>
						</nav>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<div class="inner">
							<h2>Menu</h2>
							<ul class="links">
								<li><a href="index.html">Home</a></li>
								<li><a href="research_hydrogel.html">Research 1 (in paraperation)</a></li>
								<li><a href="research_nunchucks.html">Research 2 (in paraperation)</a></li>
								<li><a href="research_pluronics.html">Research 3 (in paraperation)</a></li>
								<li><a href="index_ch.html">中文版主页(制作中)</a></li>

								<!-- <li><a href="#">Log In</a></li>
								<li><a href="#">Sign Up</a></li> -->
							</ul>
							<a href="#" class="close">Close</a>
						</div>
					</nav>

				<!-- Banner -->
					<section id="banner">
						<div class="inner">
							<!-- <div class="logo"><span class="icon fa-envelope"><a href="mailto:jy380@cam.ac.uk">jy380@cam.ac.uk</a></span></div> -->
							<h2>Dr.Yu, Jiaming</h2>
							<p>My personal portfolio - Online Curriculum Vitae</a></p>
							<span class="image left"><img src="images/me2.jpg" alt="my_photo" /></span>
							<p>Determined and studious, with a lively interest in a broad range of subjects, 
							it is bio-electronics, data-analysis, computational physics and nanotechnologies which engages my intellect most keenly. Analytically-minded and skilled in solving problems with creative
							solutions, my commitment to this area of study is total. <br> 
							</a></p>
							<p><small>“Hinc lucem et pocula sacra” 
								(From here [we derive] light and sacred draughts [literally cups]) - UNIVERSITY OF CAMBRIDGE <br> </small>

								</p>
						</div>
					</section>

				<!-- Wrapper -->
					<section id="wrapper">

						<!-- One -->
							<section id="one" class="wrapper spotlight style1">
								<div class="inner">
									<a href="https://www.cam.ac.uk/" target="_blank" rel="noopener noreferrer" class="image"><img src="images/cam.jpg" alt="" /></a>
									<div class="content">
										<h2 class="major">Education in Cambridge</h2>
										<b>2018-2022 Doctor of Philosophy in Physics (Ph.D.) </b>
										<p>Optoelectronics Group, Cavendish Laboratory <br>
										Department of Physics, University of Cambridge, United Kingdom <br>
										Supervisor:  <a href="https://scholar.google.co.uk/citations?user=ipgdVbgAAAAJ&hl=en" target="_blank" rel="noopener noreferrer">Prof. Erika Eiser</a> <br>
										Thesis Title: Multi-Level Coarse-Grained Modelling on DNA Functionalised Building Blocks <br>
										(View my PhD Thesis on<a href="https://www.repository.cam.ac.uk/handle/1810/345675" target="_blank" rel="noopener noreferrer"> Cambridge Apollo</a>) 
										(View my PhD Thesis in <a href="images/Yu_PhDThesis(jy380).pdf" target="_blank" rel="noopener noreferrer"> PDF</a>)
										<br>
										Simulation Guide Book: Daan Frenkel and Berend Smit. <i>"Understanding molecular simulation: from algorithms to applications"</i>, 
										volume 1. Elsevier, 2001. <br>
										Simulation Mentor:  <a href="https://en.wikipedia.org/wiki/Daan_Frenkel" target="_blank" rel="noopener noreferrer">Prof. Daan Frenkel</a> <br>
										</p>
										<p></p>
										<b>2016-2017 Master of Philosophy in Computational Biology (M.Phil.)</b>
										<p>Department of Applied Mathematics, University of Cambridge, United Kingdom <br>
										Supervisor:  <a href="http://www.damtp.cam.ac.uk/person/sje30" target="_blank" rel="noopener noreferrer">Prof. Stephen J. Eglen</a></p>
										MPhil in Computational Biology program aims to bring together the fields of biological, mathematical, and physical sciences with the 
										quantitative elements of contemporary biology and medicine, encompassing areas such as bioinformatics. Couses including: 
										biological image analysis, computational neuroscience, genome informatics, statistics, probability and scientific programming etc. 
										By uniting researchers and specialists from various academic departments and industry, this program fosters collaborative efforts to 
										advance research in the analysis, understanding and use of data science and artificial intelligence (AI). 
										<!--This program was created by the Cambridge Computational Biology Institute, which is now known as <a href="https://www.c2d3.cam.ac.uk/c2d3-computational-biology" target="_blank" rel="noopener noreferrer">C2D3 Computational Biology</a>. 
										The Department of Applied Mathematics and Theoretical Physics oversees the operation of this course.
										The MPhil in Computational Biology course is aimed at introducing students in the biological, mathematical and physical sciences to quantitative aspects of modern biology and medicine, including bioinformatics.  
										The course was developed by the Cambridge Computational Biology Institute (now C2D3 Computational Biology) and is run by the Department of Applied Mathematics and Theoretical Physics.
										The course brings together researchers and expertise from across the academic departments and industry to drive research into the analysis, understanding and use of data science and AI. -->
										<a href="https://www.c2d3.cam.ac.uk/c2d3-computational-biology" target="_blank" rel="noopener noreferrer" class="special">Learn more</a>
									</div>
								</div>
							</section>

						<!-- Two -->
							<section id="two" class="wrapper alt spotlight style2">
								<div class="inner">
									<a href="https://www.ox.ac.uk/" target="_blank" rel="noopener noreferrer" class="image"><img src="images/College-UG-index.jpg" alt="" /></a>
									<div class="content">
										<h2 class="major">Education in Oxford</h2>
										<b>2017-2018 Master of Science in Nanotechnology for Medicine and Health Care (M.Sc./PGcert)</b>
										<p>Department of Continuing Education, University of Oxford, United Kingdom <br>
											Supervisor: <a href="" target="_blank" rel="noopener noreferrer">Dr. Christiane Norenberg </a>; Tutor: 
											<a href="" target="_blank" rel="noopener noreferrer">Prof. Peter Dobson OBE</a> 
											<!-- <a href="https://www.queens.ox.ac.uk/people/prof-peter-dobson-obe/" target="_blank" rel="noopener noreferrer">Prof. Peter Dobson OBE</a> -->
										</p>
										Nanomedicine stands at the forefront of modern healthcare. Nanoparticles have emerged as a promising platform for drug delivery, 
										enabling the extension of the patent life of drugs while significantly enhancing their targeting capabilities and therapeutic effectiveness. 
										Nanoparticles have the potential to enhance various medical imaging techniques, and in certain cases, they can provide a combined approach 
										for both diagnosis and therapy, known as 'theranostics'. Nanotechnology is serving as a foundation for numerous innovative regenerative medicine 
										strategies, relying on artificial scaffold structures. It also presents solutions for the development of next-generation point-of-care biosensors  
										and advanced gene sequencing instruments. By harnessing the power of nanotechnology, these cutting-edge applications are revolutionizing 
										the fields of regenerative medicine, diagnostics, and genetic analysis.
										This course draws on the world-class research and teaching in nanotechnology and nanomedicine enable me to understand the principles of nanotechnology 
										and its application in medical research and clinical practice. <br>
										<a href="https://www.ox.ac.uk/admissions/graduate/courses/msc-nanotechnology-for-medicine-and-health-care" target="_blank" rel="noopener noreferrer" class="special">Learn more</a>
									</div>
								</div>
							</section>

						<!-- Three -->
							<section id="three" class="wrapper spotlight style3">
								<div class="inner">
									<a href="https://www.nottingham.ac.uk/" target="_blank" rel="noopener noreferrer" class="image"><img src="images/Cover--University-of-Nottingham.jpg" alt="" /></a>
									<div class="content">
										<h2 class="major">Education in Nottingham</h2>
										<b>2013-2016 Bachelor of Engineering with Honors in Electrical and Electronic Engineering (B.Eng)</b>
										<p>Department of Engineering, University of Nottingham, United Kingdom <br>
										Supervisor: <a href="https://www.nottingham.ac.uk/engineering/people/steve.morgan" target="_blank" rel="noopener noreferrer">Prof. Stephen Morgan</a></p>
										My Bachelor degree focuses on equipping with the essential knowledge and skills in electrical engineering, enabling me 
										to engage lab-based projects, both individually and in small teams, I developed critical problem-solving and design abilities.
										Electrical Engineering are responsible for the design and development of cutting-edge technologies used in power generation and processing as well as 
										playing a vital role in distributing electrical energy, including the implementation of 'Smart Grid' technologies for renewable energy sources. 
										This study paved the way for the advancement of Signal Processing and Communications, Power Electronic Applications and Control, Software Engineering and Programming ... etc. 
										<a href="https://www.nottingham.ac.uk/ugstudy/subject/Electrical-and-Electronic-Engineering" target="_blank" rel="noopener noreferrer" class="special">Learn more</a>
									</div>
								</div>
							</section>

						<!-- Four -->
							<section id="four" class="wrapper alt style1">
								<div class="inner">
									<h2 class="major">Research Projects</h2>
									<p>DNA nanotechnology has proliferated as investigators have increasingly
										harnessed the selectivity of DNA base pairing to form multitudes of diverse structures, with
										a wide range of applications. My researches focusing on a multi-level strategy to demonstrate new horizons of coarse-grained simulation
										systems. With the service of this multi-level coarse-grained strategy, a more realistic simulation
										system can be devoted to illustrate more reliable physical properties.</p>
									<section class="features">
										<article>
											<a href="research_pluronics.html" class="image"><img src="images/DNA_Functionalised_Pluronics(JM).png" alt="" /></a>
											<h3 class="major">DNA-Functionalised Triblock Copolymers: F108 Pluronics + DNA</h3>
											<p>Pluronics®are triblock copolymers (referred to as 'PEO-PPO-PEO') that have two hydrophilic
												polyethylene oxide (PEO) outer blocks and an inner block of polypropylene 
												oxide (PPO) that is hydrophilic at low temperatures but becomes hydrophobic upon heating. 
												These block copolymers are polymeric surfactants, commercially also known
												as Poloxamers®, and are widely used in lubrication, detergency, pharmaceuticals, foaming,
												emulsification and coating.
												In this projects, I built a coarse-grained simulation model with temperature-sensitive hydrophobic monomers to study the
												structure and phase behaviour of a symmetric bead-spring model for the PEO-PPO-PEO
												polymers. Further, we extended our model by adding a sticky patch to each free PEO
												chain, thus mimicking the complementary DNA sticky ends. This stickiness provides an
												additional, temperature dependent driving force for aggregation between two chain ends
												at low temperatures.</p>
											<a href="research_pluronics.html" class="special">Learn more</a>
										</article>
										<article>
											<a href="research_nunchucks.html" class="image"><img src="images/nunchucks_dense.png" alt=""  /></a>
											<h3 class="major">Towards New Liquid Crystal Phases of DNA Mesogens</h3>
											<p> Liquid crystals play an essential role in biology because they combine order and mobility, offering
												a basic requirement for self-organisation and structure formation in all living systems.
												The ability of dsDNA to form LC phases when it is hydrate was first demonstrated in the
												1940s. Since that time, the LC properties of aqueous suspensions of DNA have been
												extensively characterised. The geometry and physical characteristics of DNA are closely
												related to its chemical structure.
												In this research, I report on the second-level coarse-graining of the nunchucks DNA towards
												finding new liquid phase transitions. I did this work with Kit Gallagher as a day-to-day
												supervisor in his part III project. To begin with, we developed the LAMMPS code in
												single-stage, rigid-rod simulations to confirm that the analytical techniques could accurately
												recreate Onsager’s theory. Then we investigated a system of stiff rods and concentrated on
												the transition from the isotropic to the nematic phase.</p>
											<a href="research_nunchucks.html" class="special">Learn more</a>
										</article>
										<article>
											<a href="research_hydrogel.html" class="image"><img src="images/y_shape_DNA.png" alt="" /></a>
											<h3 class="major">The Role of Flexible Joints in a Linker-Mediated DNA Hydrogel</h3>
											<p>DNA hydrogels are polymeric networks made entirely from DNA. The convenient programmability
												and multi-functional tunability of DNA make it a perfect material for constructing
												a three-dimensional structure in hydrogels. DNA hydrogels are also biocompatible
												and therefore useful for biomedical applications such as bio-sensing technology, targeted
												drug delivery, 3D bio-printing, regenerative medicine and tissue engineering.
												In this research, I reported numerical work using oxDNA to test the thermodynamic stability
												and topography of Y-shaped DNA nanostars connected via rigid, linear DNA-linkers. The
												specific DNA sequences we used were those we tested in experiments to study the formation
												of DNA hydrogels as function of temperature and DNA concentration. This work was done
												in collaboration with Dr. Iliya Stoev, who did most of the experiments in this project. Our
												dynamic light scattering (DLS) measurements showed that when inserting sufficiently long
												flexible ssDNA joints between the linear linker and the Y-shaped DNA we suppress gelation.
												This motivated me to test the effect of the length of the flexible joints.</p>
											<a href="research_hydrogel.html" class="special">Learn more</a>
										</article>
									
										<article>
											<a href="research_pluronics.html" class="image"><img src="images/micelles_display.png" alt="" /></a>
											<h3 class="major">Determining the micellisation phase boundary for multiblock copolymers </h3>
											<p>	Multiblock copolymers are creative building blocks in the repertoire of self-assembled
												polymeric micelles, with practical applications in nanoscience and biotechnology.
												Amongst the many different polymeric surfactants are the symmetric ABA-type triblock
												copolymers PEO-PPO-PEO. The micellisation of block copolymers in solutions is inherently complex as the 
												critical micellisation temperature (CMT) increase
												with decreasing concentration. Even above the CMT, unimers, micelles and
												micellar aggregates coexist, and the relative proportion of each species depends strongly on
												the temperature and polymer concentration. The CMT can be determined
												experimentally by fluorescence spectroscopy, surface tension, solubilisation methods, light
												scattering and rheology. In simulations, the exact aggregation number
												of each micelle is known; hence a more direct method can be applied to investigate
												the unimer/micellar status in an equilibration period to determine the micellisation phase
												boundary.</p>
											<a href="research_pluronics.html" class="special">Learn more</a>
										</article>
									</section>
									
									<h3 class="major">Publications</h3>
									<p>	<span class="image right"><img src="images/my_thesis.jpg" alt="my_thesis"/></span>
									</p>	
									<p>1. <b>Jiaming Yu</b>. <i>"Multi-Level Coarse-Grained Modelling on DNA Functionalised Building Blocks." Apollo:</i>	  
									<a href="https://doi.org/10.17863/CAM.93097" target="_blank" rel="noopener noreferrer"> doi.org/10.17863/CAM.93097</a> (2023)
									</p>
									
									<p>2. Iliya D. Stoev, Tianyang Cao, Alessio Caciagli, <b>Jiaming Yu </b>, Christopher Ness, Ren Liu,
										Rini Ghosh, Thomas O'Neill, Dongsheng Liu and Erika Eiser. 
										<i>"On the role of flexibility in linker-mediated DNA hydrogels."</i> Soft Matter 16.4: 990-1001. 
										<a href="https://doi.org/10.1039/C9SM01398A" target="_blank" rel="noopener noreferrer">doi.org/10.1039/C9SM01398A.</a> (2020)</p>
										
									<p>3. Kit Gallagher, <b>Jiaming Yu</b>, Ren Liu, David A. King and Erika Eiser.
										"Towards New Liquid Crystal Phases of DNA mesogens."  
										<a href="https://arxiv.org/pdf/2302.03501.pdf" target="_blank" rel="noopener noreferrer">arXiv preprint arXiv:2302.03501</a> (2023) </p>

									<p>4. Ren Liu, <b>Jiaming Yu</b>, Xiaojing Tang, Rini Ghosh and Erika Eiser,
										<i>"Dynamic Light Scattering based microrheology of End-functionalised triblock copolymer solutions."</i> 
										<a href="https://www.mdpi.com/2073-4360/15/3/481" target="_blank" rel="noopener noreferrer">Polymers 15.3: 481 </a>(2022)</p>

									<p>5. <b>Jiaming Yu</b>, Eivind Bering, Ren Liu and Erika Eiser.
										<i>"Numerical study of pluronics with DNA-functionalised triblock copolymer system."</i>
										<a href="" target="_blank" rel="noopener noreferrer">link here (In paraperation)</a></p>
		 
									<p>...</p>

									<h3 class="major">Developments</h3>
									<p><a href="https://dna.physics.ox.ac.uk/index.php/OxDNA_users_and_developers_workshop" target="_blank" rel="noopener noreferrer"> 2019 oxDNA Workshop</a>, 
										Beecroft Building, Department of Physics, University of Oxford.
										<br>
									The workshop held in Oxford from September 2019. The aim of the workshop is to bring together both developers 
									and users of the oxDNA and oxRNA models not only to share current research projects, 
									but also to set the agenda for the future development of the oxDNA and oxRNA models, code and infrastructure. 
									Particular to advance the inter-operability of DNA design and simulation tools, user-friendly tools to improve oxDNA accessibility for non-expert users. 
									I was invited as a contriubtor for brain-storm and presentation of the next-generation of software developments.
									I presented a talk about 'Numerical Study on the Effect of Flexibility in DNA Building Blocks'. <br> </p>
									<br>



						<!-- Five -->
						<section id="five" class="wrapper alt style1">
							<div class="inner">
								<h2 class="major">Other Researches</h2>
								<div class="content">
									<b>March 2018 to April 2018 <br> Nanobiosensors Research Project, Begbroke Science Park, Oxford
										</b>
									<p>Department of Continuing Education, University of Oxford, United Kingdom <br>
									Project title: <a href="images/Nano2_Assign2.pdf" target="_blank" rel="noopener noreferrer">'Optical Transduction Mechanisms in DNA-based Nanobiosensors for Fluorescence Detection' </a> <br> 
									Supervisor: Dr Christiane Norenberg </p>
								</div>
								<div class="content">
									<b>June 2017 to Sep 2017 <br> Summer Research Placement, the University of Cambridge
										(Cavendish Laboratory- Biological and Soft Systems group)</b>
									<p>Department of Physics, University of Cambridge, United Kingdom <br>
									Project title: <a href="images/FUCCI_2017.pdf" target="_blank" rel="noopener noreferrer">'Automatic image analysis and extraction of cell cycle parameters in stem cells with FUCCI expression'</a> <br> 
									Supervisor: Prof. Pietro Cicuta </p>
								</div>
								<div class="content">
									<b>Sep 2015 to Sep 2016 <br> Biomedical Research Final Year Project, the University of Nottingham </b>
									<p>Department of Engineering, University of Nottingham, United Kingdom <br>
									Project title: <a href="images/FYP_BEng_2016.pdf" target="_blank" rel="noopener noreferrer">'Connection of wearable electronics to photonic textile'</a> <br> 
									Supervisor: Prof. Stephen Morgan </p>
									<!-- <embed src="images/FYP_BEng_2016.pdf" type="application/pdf" width="100%" height="600"/> -->
								</div>
								<div class="content">
									<b>Jun 2015 to Aug 2015 <br> Student Researcher, the University of Nottingham, United Kingdom
										Engineering Research Placements (ERPs) sponsored by the BP Company</b>
									<p>Department of Engineering, University of Nottingham, United Kingdom <br>
										Research topic: ‘Effects of Nano-diamonds additives as anti-friction and
										anti-wear additives for automotive oils’ <br> 
									Supervisor: Dr. Antonino La Rocca </p>
								</div>

								<h2 class="major">Honours and Rewards</h2>
								<p> <b>Oct 2018 </b>: Cambridge Trust Scholarship for Doctal of Philosophy in Physics. <br>
									<b>Sep 2018 </b>: Outstanding Graduates from the Department of Continue Education,University of Oxford. <br>
									<b>Jul 2016 </b>: Contribution to Undergraduate Studies Award to the top graduates in department. <br>
									<b>Sep 2015 </b>: Presented High Fliers Award to the top 5% of students in Faculty of Engineering. <br>
									<b>Aug 2014 </b>: Winner of an Industry Sponsored Scholarship in the University Research (£1,600). <br>
									<b>Sep 2013 </b>: Provost’s Scholarship for outstanding academic performance award to top 1.5% in division. <br>
									<b>Sep 2012 </b>: Head’s Scholarship for excellence academic performance award to top 5% in university. <br>
								</p>

								<h2 class="major">Skills</h2>
								<p> <b>Excellent PC programmer</b>: <br>
									Skilled in C language, C++, Java, Objective Oriented Programming, Embedded Software Coding. <br>
									 Keen user of algorithms method within Engineering Software Design. <br>
									 Proficient user of R/Python/Perl/Pascal/Html Languages.<br>
								</p>
								<p> <b>Interdisciplinary learner</b>: <br>
									Electrical and electronical engineering (Control system design, digital and telecommunications, power
									networks and supply, electronic construction, signal processing, professional skills and mathematical 
									techniques for electrical and electronics, VLSI design). <br>
									Computational biology (Genetics, statistics, neuroscience, genome informatics, function genomics,
									probability, population genetic analyses, structural and system biology). <br>
									Nanotechnology (Nanoscale-electronics, Nanotechnology for medical and health care, Nano-additives).<br>
								</p>
								<p> <b>Hardware:</b>: <br>
									Digital IO and Processor Design, Microcontroller, Feedback and Control Systems, Signal Processing, Analogue and Digital Communication. <br>
									<b>Software:</b>: <br>
									Photoshop, PSpice Schematics, Auto CAD, Matlab, Proficient user of emails, internets, commands and applications in Windows/MacOS/Unix/Linux systems.<br>

								</p>

								<!--<h2 class="major">Languages</h2>
								<p>Native Chinese, Proficient English. </p> -->
								<h2 class="major">Interests</h2>
								<p>Piano (Intermediate player), Tennis (2 years in university), Football and Badminton (5 years). <br>
								Sudoku, Rubik’s cube, Programing (Practiced from teenage)
								</p>
								
								<h2 class="major">University Activities</h2>
								<p>	-Winter Schoold Orgnizor for Department of Physics, Cavendish Laboratory, University of Cambridge. <br>
									-Executive committee member of Chinese Students & Scholars Association in
									Cambridge (CSSA), University of Cambridge. <br>
									-Team leader of Fundamental Science of Nanotechnology Assignment, University of Oxford. <br>
									-Student representative of the Engineering Department for the last three years;
									attended four Learning Community Forum Meetings (LCF) organized by the
									University of Nottingham.<br>
									-Completed the Summer Training Camp organized by the Department of Electrical and Electronic Engineering. <br>
									-Received Personal Service Recognition Certificate for volunteer work (University of Nottingham) <br>
									-Received outstanding student award for performance in Student Society Awards (University of Nottingham) <br>
								</p>



							</div>
						</section>
					


								</div>
							</section>
						

					</section>

				<!-- Footer -->
					<section id="footer">
						<div class="inner">
							<h2 class="major">Get in touch</h2>
							<p>Please contact me if you need more information.</p>
							<ul class="contact">
								<li class="icon solid fa-home">
									Cavendish Laboratory, University of Cambridge <br />
									JJ Thomson Ave, Cambridge, UK, CB3 0HE <br />
								</li>
								<li class="icon solid fa-phone">(+44) 079-23563496</li>
								<li class="icon solid fa-envelope"><a href="mailto:jy380@cam.ac.uk">jy380@cam.ac.uk</a></li>
								<!--<li class="icon brands fa-twitter"><a href="#">twitter.com/untitled-tld</a></li>-->
								<li class="icon brands fa-facebook-f"><a href="https://www.facebook.com/profile.php?id=100008399655939" target="_blank" rel="noopener noreferrer">facebook.com/jiaming-yu</a></li>
								<li class="icon brands fa-github"><a href="https://github.com/OE-Yu" target="_blank" rel="noopener noreferrer" >https://github.com/OE-Yu</a></li>
								<li class="icon brands fa-linkedin"><a href="https://www.linkedin.com/in/jiaming-yu-%E5%96%BB%E5%98%89%E6%98%8E-850443b2/?originalSubdomain=uk" target="_blank" rel="noopener noreferrer" >linkedin.com/jiaming-yu</a></li>
							</ul>
							<ul class="copyright">
								<li>&copy; University of Cambridges. All rights reserved.</li><li> Web Editor: Jiaming Yu </li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>